packages/mds-audit/service.ts

```diff
@@ -167,12 +168,21 @@ export async function getVehicle(provider_id: UUID, vehicle_id: string) {
     return null
   }
   const deviceStatus = (await cache.readDeviceStatus(device.device_id)) as VehicleEvent & Device
-  if (!deviceStatus) {
-    return null
+  if (deviceStatus != null) {
+    const status = EVENT_STATUS_MAP[deviceStatus.event_type as VEHICLE_EVENT]
+    const updated = deviceStatus.timestamp
+    return { ...device, ...deviceStatus, status, updated }
   }
-  const status = EVENT_STATUS_MAP[deviceStatus.event_type as VEHICLE_EVENT]
-  const updated = deviceStatus.timestamp
-  return { ...deviceStatus, status, updated }
+  log.info(
```

Comment:
This would probably be more readable with object literal syntax,
`log.info('Missing vehicle status', { provider_id, vehicle_id })`

Though I'm not sure how the log displays object literals, we might want to wrap it in a `JSON.stringify()` if it just prints out `[Object object]`

Reply:
Good call. Verified the object literal is printed correctly.